frontend/packages/eslint-config/e2e.cjs

```diff
@@ -5,6 +5,9 @@ module.exports = {
     // Useful for testing things sequentially
     'no-await-in-loop': 'off',
 
+    // Allow conditional statements in tests
+    'playwright/no-conditional-in-test': 'off',
```

Comment:
Are we ok with this change?  It looks like we ignore this rule fairly regularly anyway...